Source/Advance.cpp

```diff
@@ -91,7 +91,7 @@ PeleC::do_mol_advance(
   for (int n = 0; n < src_list.size(); ++n) {
     if (
       src_list[n] != diff_src
-#ifdef AMREX_PARTICLES
+#ifdef SPRAY_PELEC
```

Comment:
Can we use `PELEC_SPRAY` to be more consistent with others? Also, I assume the definition for this in the compiler will come in a later PR?

Reply:
Yes and yes.